web-www/package.json

```diff
@@ -22,7 +22,7 @@
     "@mdx-js/react": "^2.3.0",
     "@mui/material": "^5.10.10",
     "@mui/styles": "^5.10.10",
-    "@netlify/plugin-nextjs": "4.41.3",
+    "@netlify/plugin-nextjs": "5.7.0",
     "@next/mdx": "^13.4.19",
```

Comment:
I see this was bumped to a new major version. There are a couple breaking changes, not sure if these are related: https://docs.netlify.com/frameworks/next-js/overview/#breaking-changes

cc @clevinson

Reply:
This was changed here in #2408  https://github.com/regen-network/regen-web/pull/2408/commits/bc73188de37115ca680eb6656b96d458174a2eab It looks like all previews have been failing after this was merged ~12 hours ago.

I think going back to v4 of `netlify/plugin-nextjs` might fix the problem?